apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -16,7 +16,26 @@ defmodule AndiWeb.EditLiveView do
   def render(assigns) do
     dataset_id = assigns.dataset.id
 
+    modifier =
+      case assigns.show_unsaved_changes_modal do
+        true -> "visible"
+        false -> "hidden"
+      end
+
     ~L"""
+    <div class="unsaved-changes-modal unsaved-changes-modal--<%= modifier %>">
```

Comment:
How are we doing the other modal? Is it its own live component? Just curious if this is a similar setup to that one, to avoid bloating this file any more. If it's consistent though, that's fine.